src/index.html

```diff
@@ -6,8 +6,50 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article data-qa="card" class="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="photo of the product"
+      >
+      <div class="card__information">
+        <h2 class="card__header">
+          APPLE A1419 iMac 27" Retina <br>
+          5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+      </div>
+
+      <div class="card__review">
+        <div class="card__stars card__stars--acive">
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+        </div>
```

Comment:
Stars should be a separate BEM block, as described in requirements.
```suggestion
        <div class="stars stars--active">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```